subprojects/docs/src/docs/userguide/authoring_maintainable_builds.adoc

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-[[authoring_maintainable_build_scripts]]
-= Authoring Maintainable Build Scripts
+[[authoring_maintainable_builds]]
```

Comment:
Not that strongly, but I find inconsistencies are small, but persistent, obstacles in productivity. Also, I wasn't thinking and forgot that this would affect the URL. My bad.

Reply:
All good :). Let's keep it the same to save 2 redirects if you don't mind too much.

Reply:
er.. I mean keep it how it was in 4.10